changelog/seqera-enterprise/v25.1.mdx

```diff
@@ -0,0 +1,144 @@
+---
+title: Seqera Enterprise v25.1
+date: 2025-04-09
+tags: [seqera enterprise]
+---
+
+Seqera Platform Enterprise version 25.1 introduces Studios GA and a number of bug fixes and performance enhancements.
+
+Studios is Seqera's in-platform tool for secure, on-demand, interactive data analysis using containers created from Seqera-managed container template images or your own organization managed custom environments. You only pay for the compute your Studio sessions consume and the compute is adjacent to your data, significantly reducing data transfer costs and wasted time copying data from storage to analysis. This significantly reduces infrastructure management requirements, removes data silos, adheres to strict in-platform role-based access control, and lowers your operational costs. Learn more about Studios in our [documentation](https://docs.seqera.io/platform-enterprise/latest/studios)
+
+## New features and improvements
+
+### Studios
+
+- **Labeled templates**: Labels indicate the status of support (`recommended` or `deprecated`) for a Seqera-managed Studio container template version. Users can migrate a Studio to a new base container template when **Adding as new**.
+- **[Private sessions](https://docs.seqera.io/platform-enterprise/latest/studios/managing#general-config)**: When adding a new Studio, the number of concurrent connections (private vs. all workspace members) can now be defined in **General Config > Collaboration**.
+- **[Custom session lifespans](https://docs.seqera.io/platform-enterprise/latest/studios/managing#modify-session-lifespan-for-all-studios)**: This new feature allows you to set a maximum lifespan for a session, after which time the session is stopped automatically and a checkpoint created, or the session can be extended on-demand.
+- **Resource labels**: Users with at least workspace maintainer role permissions can manage the resource labels inherited from the compute environment and attached to the Studio. Resource labels attached to each Studio don’t affect the default resource labels associated with the compute environment.
+
+### Fusion
+
+- AWS Batch compute environments default to Amazon Linux 2023 AMI when Fusion v2 is enabled.
+- Add support for Fusion licensing.
+
+### General
+
+- Updated list of EC2 families with NVMe disk available 
+- Audit log update: Pipeline edit events are now logged.
+- Switch AWS Batch compute environment dependencies to AWS SDK v2.
+- Switch Compute dependencies to AWS SDK v2.
+- When a GitHub pipeline is added or updated, if no user-uploaded icon is defined, the icon will default to the one defined the `manifest.icon` Nextflow config; otherwise, it will use the GitHub organization avatar. You can also upload custom icons, which take priority over the auto fetched ones.
```

Comment:
```suggestion
- You can now upload custom icons when adding or updating a GitHub pipeline. If no user-uploaded icon is defined, the icon defined in the `manifest.icon` Nextflow config field is used by default. Otherwise, the GitHub organization avatar is used.
```